test-refined/shared/src/main/scala/zio/test/refined/CollectionInstances.scala

```diff
@@ -2,73 +2,108 @@ package zio.test.refined
 
 import eu.timepit.refined.api.Refined
 import eu.timepit.refined.collection.{NonEmpty, Size}
-import zio.test.Gen
-import zio.test.magnolia.DeriveGen
 import zio.{Chunk, NonEmptyChunk}
+import zio.random.Random
+import zio.test.{Gen, Sized}
+import zio.test.magnolia.DeriveGen
 
 object collection extends CollectionInstances
 
 trait CollectionInstances {
+
+  def nonEmptyChunkRefinedGen[C, T](implicit
+    genT: Gen[Random with Sized, T]
+  ): Gen[Random with Sized, Refined[NonEmptyChunk[T], NonEmpty]] =
+    nonEmptyChunkGen(genT).map(Refined.unsafeApply)
+
+  def nonEmptyListRefinedGen[T](implicit
+    genT: Gen[Random with Sized, T]
+  ): Gen[Random with Sized, Refined[List[T], NonEmpty]] =
+    nonEmptyChunkGen(genT)
```

Comment:
Maybe construct these directly instead of going through `Chunk`. `Gen.listOf1(genT).map(Refined.unsafeApply)`,